crates/wysiwyg/src/dom/unicode_string.rs

```diff
@@ -61,6 +62,42 @@ pub trait UnicodeStr:
 
     // Should really be `-> Self::Chars<'a>`, but that requires GATs
     fn chars(&self) -> Box<dyn Iterator<Item = char> + '_>;
+
+    /// Returns the length of the char in indices of the current encoding
+    fn char_len(&self, char: &char) -> usize;
+
+    /// Splits the current UnicodeStr into graphemes (visual characters), from start to end indexes
+    fn graphemes_us(&self, start: usize, end: usize) -> Vec<&Self> {
```

Comment:
Is this function used?

Reply:
No, my bad 😅 . This is the previous function, I left it there while implementing the new version and forgot to remove it.